helpers/kubernetes.go

```diff
@@ -84,33 +83,25 @@ func getK8SClientSet(kubeconfig []byte, contextName string) (*kubernetes.Clients
 	if len(kubeconfig) == 0 {
 		clientConfig, err = rest.InClusterConfig()
 		if err != nil {
-			err = errors.Wrap(err, "unable to load in-cluster kubeconfig")
-			logrus.Error(err)
-			return nil, err
+			return nil, ErrInvalidK8SConfig(err)
```

Comment:
Create a `ErrInClusterConfig` error instead using the same for external and internal config.